src/main/java/com/epam/ta/reportportal/core/item/impl/UpdateTestItemHandlerImpl.java

```diff
@@ -120,13 +126,19 @@ public class UpdateTestItemHandlerImpl implements UpdateTestItemHandler {
 
   private final Map<StatusEnum, StatusChangingStrategy> statusChangingStrategyMapping;
 
+  private final AnalyticsStrategyFactory analyticsStrategyFactory;
+  private final AnalyzerServiceClient analyzerServicesClient;
```

Comment:
do we really need analyzer here ?

Reply:
AnalyzerServiceClient is the only class which contains reuseable implementation of checking for analyzer availability. I am not sure if there are other options

Reply:
used AnalyzerServiceClient in AnalyzerManualStart class